### PR TITLE
drivers: imx: remove unnecessary blank lines

### PR DESCRIPTION
**Pull request by alaa-emad**
Remove unnecessary blank lines before a close brace as reported by checkpatch script

"CHECK: Blank lines aren't necessary before a close brace '}'"

### CODE REVIEW

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Review by dbaluta — APPROVED** [review]

**Review by paulstelian97 — APPROVED** [review]
I have zero clue how these were introduced in the first place. Thanks.

**Review by kv2019i — APPROVED** [review]
Thank you @alaa-emad !

**Comment by iuliana-prodan** [comment]
Thank you @alaa-emad!

**Comment by alaa-emad** [comment]
> Thank you @alaa-emad !

Thank you

**Review by lyakh — APPROVED** [review]

**Comment by lyakh** [comment]
> I have zero clue how these were introduced in the first place. Thanks.

@paulstelian97 it was some scripted patching, where some code lines were removed, but it was more difficult to also remove empty lines around those lines

File: src/drivers/imx/sdma.c [hunk 1]
```diff
@@ -443,6 +443,7 @@ static void sdma_disable_event(struct dma_chan_data *channel, int eventnum)
 static void sdma_channel_put(struct dma_chan_data *channel)
 {
 	struct sdma_chan *pdata = dma_chan_get_data(channel);
+
```
Comment (paulstelian97) [inline src/drivers/imx/sdma.c:446]: This is not an unnecessary blank line. It's a blank line after declaration, it should stay in.

**Comment by dbaluta** [comment]
@alaa-emad can you please the spelling error in commit message:

WARNING: 'blanck' may be misspelled - perhaps 'blank'?

**Comment by lgirdwood** [comment]
test this please

**Comment by lgirdwood** [comment]
SOFCI TEST

**Review by dnikodem — APPROVED** [review]

**Comment by kv2019i** [comment]
Only Intel CI checks failing. As this PR does not touch files that affect Intel builds, it's safe to ignore these checks, not related to this PR.

**Pull request merged**